src/Routes/Devices/UpdateSystem.js

```diff
@@ -234,7 +247,7 @@ const UpdateSystemMain = ({
     const pathLen = destPath.split('/').length;
     const dest = pathname.split('/').slice(0, pathLen).join('/');
     if (navigateProp) {
-      navigate({ pathname: dest });
+      navigate(paths.insightsInventory);
     } else {
```

Comment:
@mgold1234 what did this change break before?

Reply:
this change break the navigation after performing update to system it's open system details page instead of system table

Reply:
@mgold1234 the issue is when i update system from Host's details it's different for Edge Management page and Insight Inventory:

- in Edge Management when i open system's details and start to update system - i will be redirected back to System's details

- in Inventory when i open system's details and start to update system - i will be redirected back to Immutable tab instead of system's details

1. should it be same as in Edge Management? Maybe we should ask UX designer
2. if it's should be same as Edge Management, @de1987 i think this change should be done in insights-inventory-frontend repo, not here

Reply:
@computercamplove https://github.com/RedHatInsights/insights-inventory-frontend/pull/2032#issuecomment-1740534230
I understood it should redirect to Host's details as you mentioned in the PR from the link above

copying your quote
> 1. when I'm in process to update edge system, after i click "Update system" button it redirect me to preview/insights/inventory/ - it should me redirect back to host's details